teachertool/src/components/RubricWorkspace.tsx

```diff
@@ -87,6 +108,7 @@ const WorkspaceToolbarButtons: React.FC = () => {
     const actionItems = getActionMenuItems(activeTab);
 
     const onAutorunChange = (checked: boolean) => {
+        pxt.tickEvent(Ticks.Autorun, { checked: checked ? "true" : "false" });
```

Comment:
I removed a different tick because it seemed extraneous, but I'd like to keep this one. I don't imagine teachers will be toggling it a lot, but I am curious to see how often it is toggled, and which toggle state is more popular over time.

Reply:
Sounds good!